thrift0.13/src/main/java/com/linecorp/armeria/internal/common/thrift/ThriftFunction.java

```diff
@@ -375,4 +372,30 @@ private static String typeName(Type type, Class<?> funcClass, String methodName,
 
         return funcClassName.substring(0, serviceClassEndPos) + '$' + toAppend;
     }
+
+    /**
+     * Convert method name to LowCamel by algorithm in thrift java compiler.
```

Comment:
Could you add the link to the source code you referred, please?

Reply:
OK